fatoptimizer/methods.py

```diff
@@ -24,6 +24,14 @@ def check_encoding(args):
     return True
 
 
+def check_bytetype(args):
+    return all(isinstance(arg, bytes) for arg in args)
+
+
+def check_byte_or_int(*args):
+    return all(any([isinstance(arg, bytes), (isinstance(arg, int) and 0 <= arg <= 255)]) for arg in args)
```

Comment:
The usage of any() is strange here. I would prefer to have a simple "test1 or test2".

Please also write "for arg in args" on a second line to make the line shorter and so make the code easier to understand.